examples/src/instantiation.js

```diff
@@ -40,6 +40,7 @@ export function getInstance() {
     },
   ];
 
+  // const address = '0xbAB4d9fAaCCE0764d8663137aD14B2F9933C0Cb7';
```

Comment:
I think this commented line was missed? Was the goal to update the address below only?
Suggest removing it and write only the write address for the constant